syft/formats/spdxtagvalue/encoder_test.go

```diff
@@ -115,7 +115,7 @@ func redactor(values ...string) testutils.Redactor {
 				`Created: .*`: "Created: redacted",
 
 				// each SBOM reports a unique documentNamespace when generated, this is not useful for snapshot testing
-				`DocumentNamespace: https://anchore.com/syft/.*`: "DocumentNamespace: redacted",
+				`DocumentNamespace: https://anchore.com/.*`: "DocumentNamespace: redacted",
```

Comment:
are we intentionally changing the behavior to not include syft here?

Reply:
yes, it's a very specific field which is only set by syft when run as syft